string data_path
string mesh_file
geometry_msgs/PoseStamped[] extrinsics
---
bool success
